Dockerfile

```diff
@@ -23,6 +23,7 @@ COPY --from=app-stage ldes-server-infra-mongo/mongock-config/target/mongock-conf
 COPY --from=app-stage ldes-server-infra-mongo/mongock-changeset-1/target/mongock-changeset-1-jar-with-dependencies.jar ./lib/
 COPY --from=app-stage ldes-server-infra-mongo/mongock-changeset-2/target/mongock-changeset-2-jar-with-dependencies.jar ./lib/
 COPY --from=app-stage ldes-server-infra-mongo/mongock-changeset-3/target/mongock-changeset-3-jar-with-dependencies.jar ./lib/
+COPY --from=app-stage ldes-server-infra-mongo/mongock-changeset-5/target/mongock-changeset-5-jar-with-dependencies.jar ./lib/
```

Comment:
What happened to 4?

Reply:
4 is already on its way in https://github.com/Informatievlaanderen/VSDS-LDESServer4J/pull/533 :) But I am waiting to merge that PR at the start of a sprint.